src/adapter/src/coord.rs

```diff
@@ -538,8 +538,9 @@ impl<S: Append + 'static> Coordinator<S> {
                 CatalogItem::Sink(sink) => {
                     // Re-create the sink on the compute instance.
```

Comment:
this seems outdated now, because `CatalogItem::Sink` is only meant for "real" sinks, right?